fractals/sunqua-peak/index.md

```diff
@@ -51,6 +49,12 @@ Do the mini jumping puzzle to the 2nd boss, remember to talk to the elementals.
 
 </GridItem>
 
+<GridItem sm="12">
+However, the waterfalls have two states, active and inactive. When inactive, they will not push you over the edge. An easy way to tell which state they are in is if the water flow is more intense with more white streaks running through it or not. When the effects expire and the flow slows down, it's safe to simply walk through. Alternativly, you can also try and dodge through the intense flow or use a movement skill to quickly get past. The second waterfall can be easily skipped by walking behind it. 
```

Comment:
From my experiance using choya and skill 2 while stick to the waterfall start did also get me through "active" one pretty ez